force-app/main/default/flows/Before_Save_Session_Registration_Validations.flow-meta.xml

```diff
@@ -104,19 +117,126 @@
             <label>Yes</label>
         </rules>
     </decisions>
+    <decisions>
+        <name>Speaker_Overlapping</name>
+        <label>Speaker Overlap</label>
+        <locationX>270</locationX>
+        <locationY>671</locationY>
+        <defaultConnector>
+            <targetReference>Speaker_Reg_Loop</targetReference>
+        </defaultConnector>
+        <defaultConnectorLabel>Default Outcome</defaultConnectorLabel>
+        <rules>
+            <name>Overlapping</name>
+            <conditionLogic>(1 AND 2) OR (3 AND 4)</conditionLogic>
```

Comment:
@bmtemakina @sgoEma  I'm not sure this condition logic covers all combinations/conditions, did you get a chance to check ?